diff -u -rN include/webvtt/util.h include/webvtt/util.h
--- include/webvtt/util.h	2013-02-01 17:08:39 -0500
+++ include/webvtt/util.h	2013-02-01 17:10:00 -0500
@@ -5,14 +5,15 @@
 extern "C" {
 #endif

+# ifndef WEBVTT_STATIC
+#   define WEBVTT_STATIC (1)
+# endif
+
 # if defined(_WIN32) || defined(__WIN32__) || defined(__TOS_WIN__) || defined(__WINDOWS__)
-#   include "webvtt-config-win32.h"
 #   define WEBVTT_OS_WIN32 1
 #   if defined(_WIN64)
 #     define WEBVTT_OS_WIN64 1
 #   endif
-# else
-#   include <webvtt/webvtt-config.h>
 # endif

 # if defined(_MSC_VER)
diff -u -rN ../webvtt2/src/Makefile.in ./src/Makefile.in
--- src/Makefile.in	1969-12-31 19:00:00 -0500
+++ src/Makefile.in	2013-02-01 17:11:02 -0500
@@ -0,0 +1,16 @@
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+DEPTH           = ../..
+topsrcdir       = @top_srcdir@
+srcdir          = @srcdir@
+VPATH           = @srcdir@
+
+include $(DEPTH)/config/autoconf.mk
+
+MODULE            = webvtt
+LIBRARY_NAME      = webvtt
+
+# Empty makefile to force directory creation
+
+include $(topsrcdir)/config/rules.mk
